Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsDatabaseStore.h

```diff
@@ -166,6 +168,7 @@ class ResourceLoadStatisticsDatabaseStore final : public ResourceLoadStatisticsS
         unsigned domainID;
         RegistrableDomain registrableDomain;
         WallTime mostRecentUserInteractionTime;
+        WallTime mostRecentWebPushTime;
```

Comment:
It feels like this name and its associated functions lack some detail. The standard is called WebPush, a push can be called a WebPush, a visible push can be called a WebPus, and a push notification interacted with can be called WebPush. Could we add detail so that it's clear what this timestamp catches?